test/pg_hba.conf.in

```diff
@@ -3,14 +3,14 @@
 # "local" is for Unix domain socket connections only
 local   all             all                                                         trust
 # IPv4 local connections:
-hostssl  all             @TEST_ROLE_CLUSTER_SUPERUSER@  127.0.0.1/32           cert clientcert=1
-hostssl  all             @TEST_ROLE_1@                  127.0.0.1/32           cert clientcert=1
-hostssl  all             @TEST_ROLE_2@                  127.0.0.1/32           cert clientcert=1
+host     all             @TEST_ROLE_CLUSTER_SUPERUSER@  127.0.0.1/32           trust
```

Comment:
> Rebased again - I had to change the pg_hba.conf.in file that's used by regression tests to remove references to SSL, otherwise I couldn't run the tests locally (when compiling Postgres and TimescaleDb without SSL support). Was it intentional to break no-SSL support for the tests?

> Similarly, I'm unable to run the TSL tests on Postgres builds without SSL support because of the postgresql.conf.in file in tsl/test directory that sets ssl=on. I haven't changed that in the PR though, as changing that also changed the port number that data nodes were running on, which in turn changes a lot of the expected test output.

I cannot run test cases locally on Postgresql builds that are built without SSL support. I would actually expect that when I build TimescaleDb without SSL, that I could then run the tests on a Postgres server without SSL support (why else would I build TimescaleDb without SSL support). I'm ok with changing it back, but I think the implications of this should be clear.

Reply:
We'd like to run tests with SSL since that will be our recommended setup for multinode configurations. The testing will be incomplete without it. Still, since this is a generated file, it should be possible to select auth method based on whether SSL is enabled in CMake or not and generate a working pg_hba.conf.

Reply:
I'll revert the change here then